### ISSUE TITLE
Rename Session Prompt Box - Design & Behavior

### ISSUE THREAD

**Issue by socall** [omerxx/tmux-sessionx#89]
My issues with the Rename Session Prompt Box:
1. takes a lot of space, have no borders or fill that distinguish it from the bg.
2. no obvious way to escape it if you wish to not rename the session (I suggest pressing escape to go back to SessionX)

<img width="1440" alt="image" src="https://github.com/omerxx/tmux-sessionx/assets/5202056/af003ecc-fa1a-4d2c-be0f-a97c6c52baae">

**Comment by omerxx** [comment]
@socall I agree with the input, my initial hope was to go with `tmux display-popup` instead of an alternate shell screen but I couldn't get it to work.
If you have an idea for either an `fzf` feature or a popup that can work from within the alternate shell I'd be happy to give this another go

**Comment by timsofteng** [comment]
@omerxx Is it impossible to use tmux native rename in command line? Pressing shortcut in popup brings you to tmux's command line with rename command.

**Comment by omerxx** [comment]
@timsofteng Not sure I follow, how would you reach the Tmux prompt from within an fzf popup?
